### ISSUE TITLE
Text/code in figures can cause alt text failure

### ISSUE THREAD

**Issue by pnb** [pnb/paper-convert-scripts#15]
Example: FtUS51HHCI

There is some `<pre>` code included as part of the figure, and I expect the \Description command ends up matching it to the wrong thing.

**Comment by pnb** [comment]
This paper seems to do much worse with the latest make4ht
